Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM docker.io/centos:centos8
 
-ARG PKGS_LIST=main-packages-list.txt
+ENV PKGS_LIST=main-packages-list.txt
+ARG EXTRA_PKGS_LIST
 
```

Comment:
Not a problem with this PR but 
"ARG PATCH_LIST" 
is missing here

Reply:
bad rebase, I'll correct it

Reply:
oh wait, I see what you mean, it's actually missing from the other patch!
I'll fix that